engine/src/main/java/org/terasology/rendering/openvrprovider/OpenVRProvider.java

```diff
@@ -63,6 +63,7 @@
     //keyboard
     private static boolean keyboardShowing;
     private static boolean headIsTracking;
+    private static OpenVRProvider instance;
 
     public OpenVRState vrState = new OpenVRState();
```

Comment:
Whoa. I almost missed this one. How come this is public? Shouldn't it be readonly via an accessor and perhaps even final?